Modules/Core/Controllers/CustomFieldsController.php

```
@@ -13,39 +20,82 @@ public function __construct(CustomFieldService $customFieldService)
 
+    /**
+     * Delete a custom field.
+     *
+     * @param int $id Custom field ID
+     *
+     * @return \Illuminate\Http\RedirectResponse
+     *
+     * @legacy-function delete
+     * @legacy-file application/modules/custom_fields/controllers/Custom_fields.php
+     */
     public function delete(int $id): \Illuminate\Http\RedirectResponse
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)